src/Sarif.Multitool/FileWorkItemsOptions.cs

```diff
@@ -9,10 +10,40 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
     public class FileWorkItemsOptions
     {
         [Option(
-            'f',
+            't',
             "filing-target-type",
             HelpText = "The type of system to which work items are to be filed. Must be one of 'github' and 'azureDevOps'.",
             Required = true)]
-        public string TargetType { get; internal set; }
+        public string FilingTargetType { get; internal set; }
+
+        [Option(
+            'p',
+            "project-uri",
+            HelpText = "The absolute URI of the project under which the work items are to be filed.",
+            Required = true)]
+        public string ProjectUriString { get; internal set; }
+
+        // For an explanation of the difference between ProjectUriString and ProjectUri, see FileWorkItemsCommand.ValidateOptions.
+        public Uri ProjectUri { get; internal set; }
+
+        [Option(
+            'f',
+            "filtering-strategy",
+            HelpText = "Specifies the strategy for selecting which results to file. Must be one of 'new' and 'all'.",
+            Required = true)]
+        public string FilteringStrategy { get; internal set; }
+
+        [Option(
+            'g',
+            "grouping-strategy",
+            HelpText = "Specifies the strategy for grouping grouping SARIF results into sets that should each be filed together as a single work item. Must be 'perResult'.",
+            Required = true)]
+        public string GroupingStrategy { get; internal set; }
+
+        [Value(0,
+            MetaName = "<inputFile>",
```

Comment:
> MetaName = "<inputFile>", [](start = 11, length = 26)

Note to self and team: I don't know if we're properly utilizing MetaName across the multitool, w/be a nice clean-up if not. Great onboarding work for Surbhi or anyone else interested in ramping up over here.